src/components/Game.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react'
+import Board from './Board'
+
+//component
+const Game: React.FC = () => {
+  const initialBoard = ['', '', '', '', '', '', '', '', '']
+  const [history, setHistory] = useState<string[][]>([initialBoard])
+  const [isXNext, setIsXNext] = useState<boolean>(true)
+  const status = 'Next Player: ' + (isXNext ? 'X' : 'O')
+
+  /**
+   * based on the range value, sets the game to history[n]
+   * @param n
+   */
+  function jumpTo(n: number) {
+    setIsXNext(n % 2 === 0)
+    const newHist = history.slice(0, n + 1)
+    setHistory(newHist)
```

Comment:
```suggestion
    setHistory(history.slice(0, n + 1))
```

I'd just combine this into a one-liner